std/algorithm.d

```diff
@@ -6626,9 +6626,9 @@ struct Until(alias pred, Range, Sentinel) if (isInputRange!Range)
     private bool predSatisfied()
     {
         static if (is(Sentinel == void))
-            return unaryFun!pred(_input.front);
+            return cast(bool) unaryFun!pred(_input.front);
         else
-            return startsWith!pred(_input, _sentinel);
+            return cast(bool) startsWith!pred(_input, _sentinel);
```

Comment:
Question I wasn't sure about: is there any difference between `e ? true : false` and `cast(bool) e`? Is the cast okay here?